src/Workspaces/Remote/ServiceHub.CoreComponents/CoreComponents.Shared.targets

```diff
@@ -145,7 +145,7 @@
         two would eventually be in sync. But we'd need to include those binaries during the out-of-sync period.
         Note, for the same reasone, we can't safely exclude shared dependencies from ServiceHub host folder.
       -->
-      <_PublishRuntimeLibraries>true</_PublishRuntimeLibraries>
+      <_PublishRuntimeLibraries>false</_PublishRuntimeLibraries>
```

Comment:
💡 Can you verify correctness of the preceding comment?

Reply:
Yes, from the RPS log, I confirmed the loading of binaries like S.C.I and S.R.M are now loaded from the .NET 8 runtime folder in VS